moto/organizations/exceptions.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import unicode_literals
+
+import json
+from werkzeug.exceptions import BadRequest
+
+
+class InvalidInputError(BadRequest):
```

Comment:
This should be subclassed from the moto `JsonRESTError`.

Please see https://github.com/spulec/moto/blob/master/moto/config/exceptions.py#L2 for an example.

Reply:
ah ok, I just made it similar to the Athena one, but no problem to change it.

Reply:
Oh! I would imagine that the athena one is broken then (since it wouldn't return the proper exception back to boto).